src/environment/Config.cpp

```diff
@@ -29,7 +29,7 @@ Config::Config(const std::string& configFile) {
     fs::ifstream configStream(fs::u8path(configFile));
 
     if (!configStream) {
-        throw std::runtime_error(std::string("Couldn't read config file ") + configFile);
+        throw std::runtime_error(std::string("Couldn't read config file ") + fs::u8path(configFile).c_str());

```

Comment:
The previous version was correct: `u8path` should only be used to pass the result to `fs::ifstream` constructors, it changes the data type to `wstring` depending on the platform. If a path is only used to construct a string for the user, it's totally fine to juse use the string as-is.